GameServer/Logic/Chatbox/Commands/RainbowCommand.cs

```diff
@@ -0,0 +1,49 @@
+﻿using ENet;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using LeagueSandbox.GameServer.Logic.GameObjects;
+using static LeagueSandbox.GameServer.Logic.Chatbox.ChatboxManager;
+using LeagueSandbox.GameServer.Logic.Packets;
+
+namespace LeagueSandbox.GameServer.Logic.Chatbox.Commands
+{
+    class RainbowCommand : ChatCommand
+    {
+        public RainbowCommand(string command, string syntax, ChatboxManager owner) : base(command, syntax, owner) { }
+
+        private static Champion me;
+        private static ChatboxManager owner;
+
+        public override void Execute(Peer peer, bool hasReceivedArguments, string arguments = "")
+        {
+            me = _owner.GetGame().GetPeerInfo(peer).GetChampion();
+            owner = _owner;
+            Task.Run(() => RAINBOW());
+        }
+
+        public void RAINBOW()
+        {
+            while (true)
+            {
+                try
+                {
+                    //sending the actual Thread of execution to sleep X milliseconds
+                    float a = 0.5f;
```

Comment:
Why do you re-declare this variable every time the loop occurs? Shouldn't this be moved outside the loop?